charts/sda-svc/templates/doa-deploy.yaml

```diff
@@ -146,7 +146,7 @@ spec:
         - name: CRYPT4GH_PRIVATE_KEY_PASSWORD_PATH
           value: "{{ template "c4ghPath" . }}/passphrase"
         - name: OPENID_CONFIGURATION_URL
-          value: "{{ .Values.global.elixir.oidcdHost }}.well-known/openid-configuration"
+          value: "{{ .Values.global.elixir.oidcdHost }}/.well-known/openid-configuration"
```

Comment:
This change to DOA should probably not be part of this PR

Reply:
ok, will remove